srcpkgs/oblibs-devel

```diff
@@ -0,0 +1 @@
+oblibs/
```

Comment:
remove the `/` to match all other packages.

Reply:
Done, thanks for the review.